src/Http/Search/SearchQuery.php

```diff
@@ -0,0 +1,247 @@
+     */
+    private $q;
+
+    public function getStreet(): string
+    {
+        return $this->street;
+    }
+
+    public function setStreet(string $street): SearchQuery
+    {
+        $this->street = $street;
+
+        return $this;
+    }
+
+    public function getHouseNumber(): string
+    {
+        return $this->houseNumber;
+    }
+
+    public function setHouseNumber(string $houseNumber): SearchQuery
+    {
+        $this->houseNumber = $houseNumber;
+
+        return $this;
+    }
+
+    public function getKvkNumber(): string
+    {
+        return $this->kvkNumber;
+    }
+
+    public function setKvkNumber(string $kvkNumber): SearchQuery
+    {
+        $this->kvkNumber = $kvkNumber;
+
+        return $this;
+    }
+
+    public function getBranchNumber(): string
+    {
+        return $this->branchNumber;
+    }
+
+    public function setBranchNumber(string $branchNumber): SearchQuery
+    {
+        $this->branchNumber = $branchNumber;
+
+        return $this;
+    }
+
+    public function getRsin(): int
+    {
+        return $this->rsin;
+    }
+
+    public function setRsin(int $rsin): SearchQuery
+    {
+        $this->rsin = $rsin;
+
+        return $this;
+    }
+
+    public function getPostalCode(): string
+    {
+        return $this->postalCode;
+    }
+
+    public function setPostalCode(string $postalCode): SearchQuery
+    {
+        $this->postalCode = $postalCode;
+
+        return $this;
+    }
+
+    public function getCity(): string
+    {
+        return $this->city;
+    }
+
+    public function setCity(string $city): SearchQuery
+    {
+        $this->city = $city;
+
+        return $this;
+    }
+
+    public function isIncludeInactiveRegistrations(): bool
+    {
+        return $this->includeInactiveRegistrations;
+    }
+
+    public function setIncludeInactiveRegistrations(bool $includeInactiveRegistrations): SearchQuery
+    {
+        $this->includeInactiveRegistrations = $includeInactiveRegistrations;
+
+        return $this;
+    }
+
+    public function isRestrictToMainBranch(): bool
+    {
+        return $this->restrictToMainBranch;
+    }
+
+    public function setRestrictToMainBranch(bool $restrictToMainBranch): SearchQuery
+    {
+        $this->restrictToMainBranch = $restrictToMainBranch;
+
+        return $this;
+    }
+
+    public function getSite(): string
+    {
+        return $this->site;
+    }
+
+    public function setSite(string $site): SearchQuery
+    {
+        $this->site = $site;
+
+        return $this;
+    }
+
+    public function getContext(): string
+    {
+        return $this->context;
+    }
+
+    public function setContext(string $context): SearchQuery
+    {
+        $this->context = $context;
+
+        return $this;
+    }
+
+    public function getQ(): string
+    {
+        return $this->q;
+    }
+
+    public function setQ(string $q): SearchQuery
+    {
+        $this->q = $q;
+
+        return $this;
+    }
+
+    public function get(): array
```

Comment:
This method exposes the internal variable names. This makes it harder to rename variable names. Do we really need to expose this? If so can we do it in another way?

Like
```php
return [
    'context' => $this->getContext(),
   'query' => $this->getQ(),
];
```

Reply:
solved